src/pathfinding_service/cli.py

```diff
@@ -47,6 +47,19 @@
     help="Number of block confirmations to wait for",
 )
 @click.option("--enable-debug", default=False, is_flag=True, hidden=True)
+@click.option("--pfs-operator", default="John Doe", type=str, help="Name of the service operator")
+@click.option(
+    "--pfs-settings",
+    default="Raiden Default Settings - bidirectional Dijkstra",
+    type=str,
+    help="Information on technical implementation",
+)
+@click.option(
+    "--pfs-message",
+    default="This is your favorite pathfinding service",
```

Comment:
Can we make this more meaningful? Like `PFS for token network registry 0xdeadbeef`, or is this to technical?